.circleci/config.yml

```diff
@@ -279,31 +278,15 @@ commands:
       - run:
           name: install pip dependencies
           command: scripts/install-tests.sh
-      - run:
-          name: Install postgresql client
-          command: |
-            sudo rm -rf /var/lib/apt/lists/*
-            sudo apt update
-            sudo apt install -y postgresql-client
-      - run:
-          name: decrypt
-          command: |
-            sudo apt install openssl -y
-            openssl aes-256-cbc -d -in circle_ci_test_data.zip.enc -k $CIRCLE_CI_KEY -iv $CIRCLE_CI_IV >> test_data.zip
-            bash expand_confidential_bundle.sh test_data.zip abandonDatabaseConnectionsWithOrcid            
       - run:
           name: expand confidential data
           command: |
             bash expand_confidential_bundle.sh test_data.zip abandonDatabaseConnectionsWithOrcid 
```

Comment:
I must have missed this in the previous review, but do the instructions for updating the confidential file mention this file? Or maybe a comment in encrypt.sh

Digression: it would be nicer if the zip didn't have the nested directory, then you wouldn't need to specify the folder as a second param. But that's way out of scope, and there are possibly issues doing it that way.

Reply:
part of 2nd PR and yes, i've actually gotten rid of the expand_confidential file script already